src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_Image.cs

```diff
@@ -301,6 +301,76 @@ public async Task When_Image_Source_Nullify()
 				ExpectedPixels.At(centerX, centerY).Named("center without image").Pixel(Colors.White));
 		}
 
+		[TestMethod]
+		[RunsOnUIThread]
+#if NET461 || __MACOS__
```

Comment:
@jeromelaban  this doesn't seem to be working. How can I  skip this test for NET461 since the `MonochromaticColor` Property for the image is not available there.

Reply:
@ajpinedam The CI failure is not for NET461. It's for uap10.0.18362.

You need to wrap the whole test method with `#if !WINDOWS_UWP`